packages/angular/projects/cds-angular/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@cds/angular",
+  "version": "@VERSION",
+  "description": "Core component modules for Clarity Angular",
+  "keywords": ["ng-add", "clarity", "angular", "components", "core"],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:vmware/clarity.git"
+  },
+  "peerDependencies": {
+    "@angular/common": "^11.0.0",
```

Comment:
We should soften this version if possible, I don't know how far back we could go? It might be possible to support back to v4?